tests/apis.js

```diff
@@ -356,17 +575,18 @@ test('APIs: Types', t => {
   t.true(isEqual(apiTypes.keys, keys), 'Keys are as expected');
 });
 
-test('API: All', t => {
-  return api.all({}).then(results => {
+test.serial('API: All', t => {
+  return api.all({}, allTypes).then(results => {
     t.true(results.hasOwnProperty('items'), 'Has Items');
     t.true(results.hasOwnProperty('pages'), 'Has Pagination');
     t.true(results.items.length >= generated, 'Has at least all items in it');
   });
 });
 
-test('API: All with Follow', t => {
+test.serial.skip('API: All with Follow', t => {
```

Comment:
Need a test for `follow` with no depth query (this one as it currently exists) and one where there is a `depth=1` to show they're different, and a `depth=(>1)` to show the recursion.